helpers/mathjax.js

```diff
@@ -188,14 +189,16 @@ export function loadMathJax(mathJaxConfig) {
 				//  renders the document.  The MathDocument is returned in case
 				//  you need to rerender the shadowRoot later.
 				//
-				window.MathJax.typesetShadow = function(root, elem) {
+				window.MathJax.typesetShadow = async function(root, elem) {
 					const InputJax = startup.getInputJax();
 					const OutputJax = startup.getOutputJax();
 					const html = mathjax.document(root, { InputJax, OutputJax });
 
 					if (elem) html.options.elements = [elem];
 
-					html.render().typeset();
+					await mathjax.handleRetriesFor(() => html.render());
```

Comment:
`handleRetriesFor` is MathJax's way of enabling asynchronous actions. If it tries to render and something isn't loaded, it will retry after that loading has finished.